src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!--Font Awesome Link-->
+    <script src="https://kit.fontawesome.com/579b112cc6.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Roboto&display=swap" rel="stylesheet">
+    <title>To Do List</title>
+</head>
+<body>
+    <main>
+        <div class="app-container">
+            <div class="app-header">
+                <h1>To Do List</h1>
+                <i class="fa-solid fa-arrows-rotate"></i>
+            </div>
+            <div class="list-input">
+                <input type="text" name="task-input" id="task-input" class="task-input" placeholder="Add to your list...">
+                <i class="fa-solid fa-check-to-slot"></i>
+            </div>
+            <ul class="list">
+
+            </ul>
+            <input type="submit" value="Clear all completed" class="clear">
```

Comment:
Usually `type= "submit"` is for forms but here I don't see one. Kindly us `type="button"`

Reply:
Thanks! I just changed the input type from submit to button.